rmake.py

```diff
@@ -20,6 +20,9 @@ def parse_args():
     parser = argparse.ArgumentParser(description="""
     Checks build arguments
     """)
+
+    default_gpus = 'gfx906:xnack-,gfx1030,gfx1100,gfx1101,gfx1102,gfx1151,gfx1200,gfx1201'
```

Comment:
Just wanted to double-check - it looks like previously we used semicolons to separate the architecture strings, but here it's been switched to commas. Will that still work ok?

Reply:
Yep. Verified on both Linux and windows.